flink-runtime/src/test/java/org/apache/flink/runtime/checkpoint/ZooKeeperCompletedCheckpointStoreTest.java

```diff
@@ -160,9 +162,12 @@ public Void answer(InvocationOnMock invocation) throws Throwable {
 			stateStorage,
 			Executors.directExecutor());
 
-		SharedStateRegistry sharedStateRegistry = new SharedStateRegistry();
+		SharedStateRegistry sharedStateRegistry = spy(new SharedStateRegistry());
 		zooKeeperCompletedCheckpointStore.recover(sharedStateRegistry);
 
+		verify(retrievableStateHandle1.retrieveState(), times(1)).registerSharedStatesAfterRestored(sharedStateRegistry);
```

Comment:
Was the bug in the zookeeper checkpoint store or the standalone checkpoint store? Should there also be a test for the standalone checkpoint store to verify that it works now?